connector_importer/models/sources/source_csv.py

```diff
@@ -69,6 +69,7 @@ def _onchance_csv_file(self):
     @api.depends("csv_file")
     def _compute_csv_filesize(self):
         for item in self:
+            item.csv_filesize = False
```

Comment:
isn't easier to declare it in fields' default?

Reply:
I tested to be sure, but not it doesn't work.

It is mentionned here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration

> Check that all "compute" methods of non-stored computed fields assign a value in any case to the field, even if it is a falsy one.

Reply:
@sebalix yeah, sorry, I forgot to update my comment: I found it out (on my code as well ;P)